pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <Version>5.2.3</Version>
+    <Version>5.2.4-beta</Version>
```

Comment:
> we are planning on publishing the metapackage/Microsoft.Net.Native.Compiler package

Since UWP metapackage is built and published from the open, I am not aware of a release corresponding to which this updated version is going to be published in Dev15 RC timeframe.

@weshaggard What are your thoughts on this?